airflow-core/src/airflow/api_fastapi/core_api/routes/public/import_error.py

```diff
@@ -179,19 +179,21 @@ def get_import_errors(
         limit=limit,
         session=session,
     )
-    import_errors_result: Iterable[tuple[ParseImportError, Iterable[str]]] = groupby(
+    import_errors_result: Iterable[tuple[ParseImportError, Iterable]] = groupby(
         session.execute(import_errors_select), itemgetter(0)
     )
 
     import_errors = []
     for import_error, file_dag_ids in import_errors_result:
+        dag_ids = [dag_id for _, dag_id in file_dag_ids]
```

Comment:
It seems like the `file_dag_ids` is still `Iterable[str]` ?
```suggestion
        dag_ids = [dag_id for dag_id in file_dag_ids]
```

Reply:
Nop, see my comment above